hw2/04-background.js

```diff
@@ -1 +1,43 @@
 // Add your code here
+
+//reference---- https://www.codespeedy.com/how-to-change-background-color-every-seconds-in-javascript/
+//reference---- https://codepen.io/jagadeshanh/pen/YGJmmv
+//reference---- https://developer.mozilla.org/en-US/docs/Web/API/setInterval
+//reference---- https://stackoverflow.com/questions/74280613/how-to-change-background-color-randomly-in-a-javascript-function
+
+let target = document.querySelector("body");
+let toggle = document.querySelector(".btn");
+let userInput = document.querySelector("input");
+toggle.addEventListener("click", changeBgColor);
+
+setInterval(changeBgColor, 3000);
```

Comment:
the time interval should be user input here.
for example: 
let input = document.querySelector('#timeInterval');